assets/common/src/register.py

```diff
@@ -165,23 +171,19 @@ def main():
         logger.info(f"MLModel path: {mlmodel_path}")
         with open(mlmodel_path, "r") as stream:
             metadata = yaml.safe_load(stream)
-            flavors = metadata.get('flavors', flavors)
+            flavors = metadata.get("flavors", flavors)
 
     if not model_version or is_model_available(ml_client, model_name, model_version):
-        # hack to get current model versions in registry
         model_version = "1"
-        models_list = []
         try:
-            models_list = ml_client.models.list(name=model_name)
-            if models_list:
-                max_version = (max(models_list, key=lambda x: int(x.version))).version
-                model_version = str(int(max_version) + 1)
-        except Exception:
+            model_version = str(int(ml_client.models.get(name=model_name, label="latest").version) + 1)
+        except Exception as e:
```

Comment:
What if the model doesn't exist? does it return 0?

Reply:
it will go into exception and we use `1` by default